modules/ROOT/partials/configuration/textpattern_patterns.adoc

```diff
@@ -8,8 +9,8 @@ ____
 IMPORTANT: Any formats or commands used by `textpatterns` need to be registered with the editor when it is initialized. This may include enabling relevant plugins, such as the `lists` plugin.
 For information on:
 
-* Registering formats for {productname}, see: link:{baseurl}/configure/content-formatting/#formats[Content formatting options - `formats`].
-* Registering commands for {productname}, see: xref:apis/tinymce.editorcommands.adoc[{productname} APIs - EditorCommands].
+* Registering formats for {productname}, see: xref:configure/content-formatting.adoc#formats[Content formatting options - `formats`].
+* Registering commands for {productname}, see: xref:apis/tinymce.editor.adoc#addCommand[{productname} APIs - EditorCommands].
 ____
 
 ==== The default patterns for the `textpattern` plugin
```

Comment:
The following line below is not rendering correctly:
```
`{start: '*', end: '*', format: 'italic'}` - Entering text between``*`` and then pressing the *spacebar* will result in the `italic` format being applied to the text between the `*` symbols.
```
The line below may also have some issues, so it probably needs changing to something like this:
```
* `+{start: '*', end: '*', format: 'italic'}+` - Entering text between `+*+` and then pressing the *spacebar* will result in the `italic` format being applied to the text between the `+*+` symbols.
* `+{start: '**', end: '**', format: 'bold'}+` - Entering text between `+**+` and then pressing the *spacebar* will result in the `bold` format being applied.
```

Reply:
Done

Reply:
This hasn't quite been done correctly sorry Hamza. Take a look where else I added `+` in the line and also fixed the spacing, namely around `+*+`.

Reply:
Oh I see now ok

Reply:
Done correctly this time hopefully 🙂

Reply:
Yep looks good now 👍